integration_tests/test_basic.py

```diff
@@ -69,7 +69,7 @@ def test_events(cluster, suspend_capture):
 def test_minimal_gas_price(cronos):
     w3 = cronos.w3
     gas_price = w3.eth.gas_price
-    assert gas_price == 5000000000000
+    # assert gas_price == 5000000000000
```

Comment:
gas price is not fixed now.

Reply:
specifying unit would be useful

Reply:
> specifying unit would be useful

this assertion is removed, because gas_price result is not constant anymore.